### ISSUE TITLE
Lines with only whitespace

### ISSUE THREAD

**Issue by jayvdb** [bbqsrc/xmlem#5]
The following demonstrates https://github.com/bbqsrc/xml-pretty/issues/3 exists here

```diff
diff --git a/src/document.rs b/src/document.rs
index 6ca1fa8..d0dc9ae 100644
--- a/src/document.rs
+++ b/src/document.rs
@@ -539,3 +539,18 @@ impl std::str::FromStr for Document {
         Self::from_reader(std::io::Cursor::new(s))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use crate::Document;
+
+    #[test]
+    fn test_document_trailing_whitespace() {
+        let doc = Document::from_str(r#"<text>
+    Actual Output
+    </text>"#).unwrap();
+        assert_eq!(doc.to_string_pretty(), "<text>\n  \n    Actual Output\n    \n</text>\n");
+    }
+}
```

**Comment by Ltrlg** [comment]
This should be fixed by #8. Could someone with “Close” permission close this issue?

**Comment by jayvdb** [comment]
Thanks @Ltrlg
